Geometry/CaloTopology/src/HcalTopology.cc

```diff
@@ -195,18 +195,14 @@ HcalTopology::HcalTopology(HcalTopologyMode::Mode mode,
       HFSize_(kHFSizePreLS1),
       HTSize_(kHTSizePreLS1),
       CALIBSize_(kCALIBSizePreLS1),
-      numberOfShapes_(
-          ((mode == HcalTopologyMode::SLHC) || (mode_ == HcalTopologyMode::Run3) || (mode_ == HcalTopologyMode::Run4))
-              ? 500
-              : 87) {
+      numberOfShapes_(phase2() ? 500 : 87) {
```

Comment:
This does not correspond to [L82](https://github.com/cms-sw/cmssw/blob/eb00e8ccf01deca0dec95e877569ecf300535cfb/Geometry/CaloTopology/src/HcalTopology.cc#L82). Just checking: is it correct?

Reply:
It was more or less correct. Updated it with mode LHC or not